Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -0,0 +1,128 @@
+
+import Foundation
+import MapboxDirections
+import SwiftCLI
+
+
+private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
+
+class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > : Command {
+    
+    // MARK: - Parameters
+    
+    var name = "process"
+    
+    @Key("-i", "--input", description: "Filepath to the input JSON.")
+    var inputPath: String?
+    
+    @Key("-c", "--config", description: "Filepath to the JSON, containing serialized Options data.")
```

Comment:
what do these options look like? was also considering this and it would be good to make iOS and Android recognize the same config

Reply:
These are JSON-serialized `RouteOptions` or `MatchOptions` depending on command. As Minh mentioned above, that might be not always convenient, so we'll add support for other ways to provide options data.